Moviebuff/Views/BackdropCarouselView.swift

```diff
@@ -0,0 +1,37 @@
+import SwiftUI
+
+struct BackdropCarouselView: View {
+
+    let title: String
+    let movies: [Movie]
+    @ObservedObject var viewModel: MovieViewModel
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            Text(title)
+                .font(.title)
+                .fontWeight(.bold)
+                .padding(.horizontal)
+            
+            ScrollView(.horizontal, showsIndicators: false) {
+                HStack(alignment: .top, spacing: 16) {
+                    ForEach(self.movies) { movie in
+                        NavigationLink(destination: DetailsView(movie: movie, viewModel: viewModel)) {
+                            BackdropView(movie: movie)
+                                .frame(width: 272, height: 200)
+                        }
+                        .buttonStyle(PlainButtonStyle())
+                        .padding(.leading, movie.id == self.movies.first!.id ? 16 : 0)
+                        .padding(.trailing, movie.id == self.movies.last!.id ? 16 : 0)
```

Comment:
There's no need to fore unwrap here with `!`. You can use optional chaining with `?` instead, and then you are more protected from a crash.